packages/manager/modules/exchange/src/account/update/update.routing.js

```diff
@@ -18,7 +18,17 @@ export default /* @ngInject */ ($stateProvider) => {
       goBack: /* @ngInject */ ($state) => () => $state.go('^'),
 
       emailAccount: /* @ngInject */ ($transition$) => {
-        return $transition$.params().emailAccount;
+        const { emailAccount } = $transition$.params();
+
+        if (emailAccount) {
+          const emailAccountKeys = Object.keys(emailAccount);
+          emailAccountKeys.forEach((key) => {
+            const propertyValue = emailAccount[key];
+            emailAccount[key] = Number(propertyValue) || propertyValue;
```

Comment:
**suggestion**: better to use `nullish coalescing` operator because above statement would return '0'.